### DISCUSSION TITLE
Fatal Error = Der Wartezustand wurde aufgrund eines abgebrochenen Mutex beendet.

### DISCUSSION THREAD

**Discussion by moehzhaw** [SafeExamBrowser/seb-win-refactoring#841] [Q&A]
Hello,

Recently, we had an error according to the following screen-foto, see below. Our setting is:

- Moodle 4.2.6
- SEB version: Win 3.5.0
- Error occurred during the registration process
- The student continued the exam with an other device, i.e. the error was not solved during the exam
- Unfortunately, there are no log-files available

Do you have an idea/advice to avoid this in the future? Cheers.

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/111352742/53ae5c28-9a14-4988-9f97-b4a567a8da77)

**Comment by dbuechel** [comment]
This very much looks like an issue due to an improper termination of SEB, as is also explained in the Microsoft documentation:

> In the case of a system-wide mutex, an abandoned mutex might indicate that an application has been terminated abruptly (for example, by using Windows Task Manager).
>
> https://learn.microsoft.com/en-us/dotnet/standard/threading/mutexes#abandoned-mutexes

Restarting the computer should resolve this, and avoided can the issue be by always terminating SEB normally (also and especially before shutting down the computer itself).
